tasks/configure.yml

```diff
@@ -1,24 +1,78 @@
 ---
-- name: Copy environment variables file
```

Comment:
If we remove this, will things still work for non-local builds?

Reply:
yeah I suppose so.

Reply:
why?

Reply:
Just wondering because I dont want to break non local builds

Reply:
non local builds should still run fine